modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/report/CheckboxDDMFormFieldTypeReportProcessor.java

```diff
@@ -46,12 +45,10 @@ public JSONObject process(
 
 		if (Validator.isNotNull(valueString)) {
 			if (GetterUtil.getBoolean(valueString)) {
-				valueString = LanguageUtil.get(
-					value.getDefaultLocale(), "true");
+				valueString = Boolean.TRUE.toString();
 			}
 			else {
-				valueString = LanguageUtil.get(
-					value.getDefaultLocale(), "false");
+				valueString = Boolean.FALSE.toString();
 			}
 
 			updateData(
```

Comment:
@mateusgomes01 can you please check if we really need this if/else. I believe we can simply do:
```
if (Validator.isNotNull(valueString)) {
	updateData(
		ddmFormInstanceReportEvent,
		fieldJSONObject.getJSONObject("values"), valueString);
}
```

Reply:
I agree @carolmariaabb . I'll make the change. Thanks!